Selenium_pract_task/Tets_Cases/Validate_File_Download_Function-Works-As_Expected.cs

```diff
@@ -0,0 +1,30 @@
+﻿using NUnit.Framework;
+using Selenium_pract_task.Entities.AbstractEntities;
+using Selenium_pract_task.POM.EpamMainPage;
+using static Selenium_pract_task.Constants.Constants.IOConstants;
+
+namespace Selenium_pract_task.Tets_Cases
+{
+    [TestFixture]
+    public class Validate_File_Download_Function_Works_As_Expected : BaseTest
+    {
+        private string downloadedFileName = "EPAM_Corporate_Overview_Q4_EOY.pdf";
```

Comment:
According to requirements: Validate that file “[EPAM_Systems_Company_Overview.pdf](https://www.epam.com/content/dam/epam/free_library/EPAM_Systems_Company_Overview.pdf)” downloaded (**use the name of the file as a parameter**)